amf-client/shared/src/test/resources/validations/raml-inline-required.raml

```diff
@@ -0,0 +1,21 @@
+#%RAML 1.0
+title: "test api"
+
+types:
+    testSchema: |
+      {
+        "required": [
+          "option_two"
+        ]
+      }
+
+/pets:
+  get:
+    responses:
+      200:
+        description: Not Found
+        body:
+          application/json:
+            type: testSchema
+            example:
+              option_two: {}
```

Comment:
We should add more valid examples here where the property option_two is defined with different types such as a string, array, number, and already defined object. This will ensure that setting the property range with an AnyShape is working as expected.

Reply:
Would also consider trying the value null.